modules/business-text/business-text.resolver.js

```diff
@@ -0,0 +1,64 @@
+const businessTextService = require('./business-text.service');
+
+const businessTextQuery = {
+  getAllBusinessTexts: (parent, args) => businessTextService.getAllBusinessTexts(),
+
+  getBusinessTextById: async (parent, args) => {
+    try {
+      return await businessTextService.getBusinessTextById(args.id);
+    } catch (e) {
+      return {
+        statusCode: 400,
+        message: e.message,
+      };
+    }
+  },
+  getBusinessTextByCode: async (parent, args) => {
+    try {
+      return await businessTextService.getBusinessTextByCode(args.code);
+    } catch (e) {
+      return {
+        statusCode: 400,
+        message: e.message,
+      };
+    }
+  },
+};
+
+const businessTexMutation = {
```

Comment:
Here we have a typo, please rename it to `businessTextMutation`

Reply:
fixed